gigascience/gigadb-website
# Parse table content as html

# Pull request for issue: #1566

This is a pull request for the following functionalities:

* Sanitize and parse as html the content of a table column that had unparsed html tags (also, some comments have been removed)

## How to test?

Navigate to http://gigadb.gigasciencejournal.com:9170/adminDataset/admin and verify the presence of sursive in the title column

## How have functionalities been implemented?

* CHtmlPurifier used to purify data, and use 'type' => 'raw' to parse it as html
* I checked other admin tables but none of them had htlm tags. I don't think we need this parsing preemptively so didn't update any more columns

## Any issues with implementation?

None

## Any changes to automated tests?

None

## Any changes to documentation?

None

## Any technical debt repayment?

None

## Any improvements to CI/CD pipeline?

None
